docs/custom-index-template.asciidoc

```diff
@@ -67,6 +79,34 @@ Specify the number of replica shards in the **index settings**:
 }
 ----
 
+[discrete]
+==== Mappings
+
+{ref}/mapping.html[Mapping] is the process of defining how a document, and the fields it contains, are stored and indexed.
+In the *Mappings* step, you can add custom field mappings.
+For example, you could:
+
+* Add custom field mappings that you can index on and search.
+In the *Mapped fields* tab, add a new field including the {ref}/mapping-types.html[field type]:
++
+image::images/custom-index-template-mapped-fields.png[Editing a component template to add a new mapped field]
+
+* Add a {ref}/runtime.html[runtime field] that is evaluated at query time.
+In the *Runtime fields* tab, click *Create runtime field* and provide a field name,
+type, and optionally a script:
++
+image::images/custom-index-template-runtime-fields.png[Editing a component template to add a new runtime field]
+
+// * {Do something.} In the *Dynamic templates* tab...
+
+// * {Do something.} In the *Advanced options* tab...
+
+// [discrete]
+// ==== Aliases
+
+// In the *Aliases* step, ...
+// For example, you could:
+
 Changes to component templates are not applied retroactively to existing indices.
 For changes to take effect, you must create a new write index for the data stream.
 This can be done with the {es} {ref}/indices-rollover-index.html[Rollover API].
```

Comment:
I think we should add a new heading to this section so it feels more separated from the previous topic.